docker-aoi-apache/README.md

```diff
@@ -0,0 +1,19 @@
+# AOI Apache Docker Image
+
+This utilizes version 2.4 of the Apache httpd server. The configuration file is extremely basic at present and will need to be built. As such, this has been tagged as prototype for the time being.
+
+To build locally
+
+docker build -t image-name:tag .
```

Comment:
Wrap inside triple backtick for pretty formatting.
Also, use the actual command we are using, in this case 'docker build -t aoi-apache .'